index.bs

```diff
@@ -1147,19 +1147,24 @@ Similarly, when piping a {{ReadableStream}} into a {{FileSystemWritableFileStrea
 <div algorithm>
 To
 <dfn local-lt="creating a new FileSystemWritableFileStream">create a new `FileSystemWritableFileStream`</dfn>
-given a [=file entry=] |file| in a [=/Realm=] |realm|:
+given a {{FileSystemFileHandle}} |fileHandle| and a [=file entry=] |fileEntry|
+in a [=/Realm=] |realm|:
 
 1. Let |stream| be a [=new=] {{FileSystemWritableFileStream}} in |realm|.
-1. Set |stream|'s [=FileSystemWritableFileStream/[[file]]=] to |file|.
+1. Set |stream|'s [=FileSystemWritableFileStream/[[file]]=] to |fileEntry|.
 1. Let |writeAlgorithm| be an algorithm which takes a |chunk| argument
-   and returns the result of running the [=write a chunk=] algorithm with |stream| and |chunk|.
+   and returns the result of running the [=write a chunk=] algorithm with
+   |fileHandle|, |stream|, and |chunk|.
 1. Let |closeAlgorithm| be these steps:
    1. Let |closeResult| be [=a new promise=].
    1. [=Enqueue the following steps=] to the [=file system queue=]:
-      1. Let |accessResult| be the result of running |file|'s
-         [=file system entry/query access=] given "`readwrite`".
+      1. [=Assert=]: [=locating an entry=] given |fileHandle|'s
+         [=FileSystemHandle/locator=] returns a [=file entry=] that is
+         [=the same entry as=] |fileEntry|.
```

Comment:
We should be guaranteed of this since the file entry is locked and modifying operations such as `move()` and `remove()` (will) take a lock (once they're specified). Do you think this is worth `Note: `ing?

Reply:
An Assert is a good way to note things. :-)

Reply:
no longer relevant as of the latest patch